### PR TITLE
Documentation Update for Views

### PR DESCRIPTION
**Pull request by Delphinus8821**
### Prerequisites

- [x] Reviewed the [checklist](CHECKLIST.md)

- [ ] Reviewed feedback from the "Sonar Cloud" bot. Note that you have to wait
    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change
Added help buttons to the Attribute Editor, Find View and Histogram View.
Updated images and help descriptions in the Attribute Editor, Find View, Histogram View and Layers View.

### Why Should This Be In Core?
Updates to help documentation in Core.

### Applicable Issues
#340

### CODE REVIEW

File: CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/resources/attribute-editor.css [hunk 1]
```diff
@@ -30,3 +30,9 @@
 .hidden#custom {
     -fx-base:#4f7fba;
 }
+
+#options-menu {
+    -fx-background-color: #121212;
```
Comment (antares1470) [inline CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/resources/attribute-editor.css:35]: just noting that the dark scheme here has background color applied whereas the light scheme doesn't and instead has padding and color (which dark scheme does not). Do these need to be aligned?

**Review by antares1470 — COMMENTED** [review]
For the Attribute Editor page, it might be good to add something for Show/Hide Empty (since that isn't mentioned anywhere in the page)

File: CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/resources/attribute-editor.css (hunk 1 above)
Comment (Delphinus8821) [reply CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/resources/attribute-editor.css:35]: This change was based on the options menu in the Data Access View which only needed changes made on the dark mode version. The background color of light mode matches the view whereas the dark mode background was a gradient

Comment (antares1470) [reply CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/resources/attribute-editor.css:35]: Got it. Does that include the padding? I would've assumed it being added in one but not the other would mean the options menu is aligned slightly differently depending on whether you're in light or dark mode

Comment (Delphinus8821) [reply CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/resources/attribute-editor.css:36]: I've added the padding to both to keep the button the same size

**Review by Quasar985 — APPROVED** [review]

**Review by antares1470 — APPROVED** [review]

**Pull request merged**